gnomad/utils/vcf.py

```diff
@@ -670,54 +738,64 @@ def make_hist_bin_edges_expr(
     :param ht: Table containing histogram variant annotations.
     :param hists: List of variant histogram annotations. Default is HISTS.
     :param prefix: Prefix text for age histogram bin edges.  Default is empty string.
+    :param label_delimiter: String used as delimiter when making group label combinations.
+    :param include_age_hists: Include age histogram annotations.
     :return: Dictionary keyed by histogram annotation name, with corresponding reformatted bin edges for values.
     """
     # Add underscore to prefix if it isn't empty
     if prefix != "":
-        prefix += "_"
-
-    edges_dict = {
-        f"{prefix}{call_type}": "|".join(
-            map(
-                lambda x: f"{x:.1f}",
-                ht.head(1)[f"age_hist_{call_type}"].collect()[0].bin_edges,
-            )
+        prefix += label_delimiter
+
+    edges_dict = {}
+    if include_age_hists:
+        edges_dict.update(
+            **{
+                f"{prefix}{call_type}": "|".join(
+                    map(
+                        lambda x: f"{x:.1f}",
+                        ht.head(1)[f"age_hist_{call_type}"].collect()[0].bin_edges,
+                    )
+                )
+                for call_type in ["het", "hom"]
+            }
         )
-        for call_type in ["het", "hom"]
-    }
 
     for hist in hists:
 
         # Parse hists calculated on both raw and adj-filtered data
-        for hist_type in ["raw_qual_hists", "qual_hists"]:
+        for hist_type in [f"{prefix}raw_qual_hists", f"{prefix}qual_hists"]:
 
             hist_name = hist
             if "raw" in hist_type:
-                hist_name = f"{hist}_raw"
+                hist_name = f"{prefix}{hist}_raw"
 
             edges_dict[hist_name] = "|".join(
                 map(
                     lambda x: f"{x:.2f}" if "ab" in hist else str(int(x)),
                     ht.head(1)[hist_type][hist].collect()[0].bin_edges,
                 )
             )
+
     return edges_dict
 
 
 def make_hist_dict(
-    bin_edges: Dict[str, Dict[str, str]], adj: bool, label_delimiter: str = "_"
+    bin_edges: Dict[str, Dict[str, str]],
+    adj: bool,
+    dict_hists: List[str] = HISTS,
```

Comment:
not sure `dict_hists` is the best name here; maybe something like `hist_metrics`, `hist_list`, `hist_metric_list`?